src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -784,10 +788,15 @@ def exec_bulk(
         )
 
     @staticmethod
-    def _apply_default_value_for_input(inputs: Dict[str, FlowInputDefinition], line_inputs: Mapping) -> Dict[str, Any]:
+    def _apply_default_value_for_input(
+            inputs: Dict[str, FlowInputDefinition],
+            line_inputs: Mapping,
+            node_reference_flow_inputs_keys: Optional[list] = None) -> Dict[str, Any]:
```

Comment:
How about `referenced_flow_inputs_keys`? Since node_reference is another input type in `InputValueType`
```suggestion
            referenced_flow_inputs_keys: Optional[list] = None) -> Dict[str, Any]:
```

Reply:
rename "node_referenced_flow_inputs"